php/src/Docker/DockerActionManager.php

```diff
@@ -125,7 +125,11 @@ public function DeleteContainer(Container $container) {
         $url = $this->BuildApiUrl(sprintf('containers/%s', urlencode($container->GetIdentifier())));
         try {
             $this->guzzleClient->delete($url);
-        } catch (\Exception $e) {}
+        } catch (ClientException $e) {
```

Comment:
```suggestion
        } catch (\GuzzleHttp\Exception\RequestException $e) {
```

https://docs.guzzlephp.org/en/stable/quickstart.html#exceptions

Reply:
done, thanks!